packages/amplify-graphql-transformer-interfaces/src/graphql-api-provider.ts

```diff
@@ -57,6 +57,7 @@ export interface OpenIDConnectConfig {
 
 export interface LambdaConfig {
   lambdaFunction: string;
+  lambdaArn?: string;
```

Comment:
As a customer, I'd interpret this to mean that I could reuse an existing function (which is in line with your description of the change), but since the config still requires me to provide a `lambdaFunction`, I'm not sure how to deal with it. How can we clarify the intent of the optional `lambdaArn` field on this interface, and specifically how it interacts with the `lambdaFunction` field?

Reply:
Yeah, `lambdaFunction` as the name isn't super clear, I'll drop a comment that this is the expected function name (excluding optional env).

Reply:
Do we need the functionName as well when customers provide the Arn?

Reply:
It's possible we could make it optional in the future, but that'll require more tracing through the codebase here to ensure it's not used elsewhere (I don't trust the type system to complain in these packages if we make something optional that's actually required).